elks/arch/i86/config.in

```diff
@@ -13,6 +13,16 @@ mainmenu_option next_comment
 endmenu
 
 mainmenu_option next_comment
+    comment 'Select compiler to use'
+
+    choice 'Select compiler to use'		\
+	"bcc		CONFIG_COMPILER_1	\
```

Comment:
Why not `CONFIG_COMPILER_BCC` etc.?

Reply:
This was the first implementation, but the configuration scripts send errors to screen when they see the dashes in CONFIG_COMPILER_IA16-ELF-GCC